ui/app/send.js

```diff
@@ -53,9 +55,11 @@ function SendTransactionScreen () {
       amount: '0.0001', // see L544
       gasPrice: '4a817c800',
       gas: '0x7b0d',
+      gasFee: ((parseInt('0x7b0d', 16) * parseInt('4a817c800', 16)) / 1000000000).toFixed(10),
```

Comment:
Would readability be improved by moving this out into a static method, or util function?